src/rabbit_trust_store_certificate_provider.erl

```diff
@@ -0,0 +1,23 @@
+-module(rabbit_trust_store_certificate_provider).
+
+-include_lib("public_key/include/public_key.hrl").
+
+-callback list_certs(Config)
+    -> no_change | {ok, [{CertId, Attributes}]}
+    when Config :: list(),
+         CertId :: term(),
+         Attributes :: map().
+
+-callback list_certs(Config, ProviderState)
+    -> no_change | {ok, [{CertId, Attributes}]}
+    when Config :: list(),
+         ProviderState :: term(),
+         CertId :: term(),
+         Attributes :: map().
+
+-callback get_cert_data(CertId, Attributes, Config)
```

Comment:
I'd suggest renaming this to `load_cert/3`. In fact, that function already logs that it's loading a certificate in at least one implementation.